pkg/virt-handler/node-labeller/api/BUILD.bazel

```diff
@@ -1,9 +0,0 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
Is it possible to perform these removals in the first commit?
Since the replacement happens there.

Reply:
Good catch, done.